internal/locale/locales/en-us.yaml

```diff
@@ -1764,6 +1764,7 @@ update_permission_err:
 err_update_in_progress:
   other: |
     A State Tool update appears to already be running.
```

Comment:
Can we fix the split infinitive?

"A State Tool update appears to be already running." Otherwise good!

Reply:
Done.